src/lib/utils.py

```diff
@@ -562,6 +567,7 @@ def dump_manifest(contents: t.Dict, manifest_path: t.Union[Path, str]):
 
     with manifest_path.open("w", encoding="utf-8") as fp:
         if manifest_path.suffix in (".yaml", ".yml"):
+            _yaml.explicit_start = has_yaml_header  # type: ignore[assignment]
```

Comment:
Why is this annotation needed? I guess I see above that `_yaml.width` needs the same thing.

Reply:
It's the same issue as width, it thinks it is set to None. https://sourceforge.net/p/ruamel-yaml/tickets/322/